lib/Zonemaster/Backend/Config.pm

```diff
@@ -253,16 +254,13 @@ sub parse {
     if ( defined( my $value = $get_and_clear->( 'ZONEMASTER', 'age_reuse_previous_test' ) ) ) {
         $obj->_set_ZONEMASTER_age_reuse_previous_test( $value );
     }
-
-    $obj->{_LANGUAGE_locale} = {};
-    for my $locale_tag ( split /\s+/, $get_and_clear->( 'LANGUAGE', 'locale' ) || 'en_US' ) {
-        $locale_tag =~ /^[a-z]{2}_[A-Z]{2}$/
-          or die "Illegal locale tag in LANGUAGE.locale: $locale_tag\n";
-
-        !exists $obj->{_LANGUAGE_locale}{$locale_tag}
-          or die "Repeated locale tag in LANGUAGE.locale: $locale_tag\n";
-
-        $obj->{_LANGUAGE_locale}{$locale_tag} = 1;
+    if ( defined( my $value = $get_and_clear->( 'LANGUAGE', 'locale' ) ) ) {
+        if ( $value ne "" ) {
+            $obj->_clear_LANGUAGE_locale();
```

Comment:
It looks like you are emptying the hash before populating it. If this is really the case, I think the name `_clear_LANGUAGE_locale` is conflicting with `get_and_clear` (which removes an entry from the ini file), both uses the word "clear". What do you think about renaming the method to something like `_reset_LANGUAGE_locales` ?

Reply:
Good point. Fixing.